config/roots.php

```diff
@@ -10,12 +10,15 @@
     'i18n' => function (array $roots) {
         return $roots['kirby'] . '/i18n';
     },
-    'translations' => function (array $roots) {
+    'i18n:translations' => function (array $roots) {
         return $roots['i18n'] . '/translations';
     },
     'i18n:rules' => function (array $roots) {
         return $roots['i18n'] . '/rules';
     },
+    'translations' => function (array $roots) {
+        return $roots['i18n:translations'];
```

Comment:
Doesn't it have to be the other way around? Like this, it will break when the old `translations` root was overwritten because our new code only cares about the new root (which won't notice that the old root has been overwritten).